.github/workflows/zoweReleaseVersion.yaml

```
@@ -0,0 +1,78 @@
+name: Update Zowe Version and Create PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      major_version:
+        description: 'Major version (default is 3)'
+        required: true
+        default: '3'
+      minor_version:
+        description: 'Minor version'
+        required: true
+      patch_version:
+        description: 'Patch version'
+        required: true
+
+jobs:
+  update_versions_and_create_pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout CLI Repo
+        uses: actions/checkout@v4
+        with:
+          repository: 'zowe/zowe-cli'
+          ref: ${{ env.defaultBranch }}
+          token: ${{ secrets.ZOWE_ROBOT_TOKEN }}
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: lts/*
+
+      - name: Set Variables and Ensure Increase in Semver
+        id: get_zowe_versions
+        run: |
+          current_version=$(jq -r '.zoweVersion' packages/cli/package.json)
+          current_version="${current_version#V}"  # Strip the 'V' prefix
+          echo "current_version=$current_version" >> $GITHUB_ENV
+          new_version="${{ github.event.inputs.major_version }}.${{ github.event.inputs.minor_version }}.${{ github.event.inputs.patch_version }}"
+          echo "new_version=$new_version" >> $GITHUB_ENV
+          echo "PACKAGE_JSON_PATH=packages/cli/package.json" >> $GITHUB_ENV
+          if [[ "$new_version" < "$current_version" ]]; then
+            echo "Error: New version $new_version is not greater than current version $current_version."
+            exit 1
+          fi
+
+      - name: Create Branch
+        run: |
+          branch_name="update-version-V${{ env.new_version }}"
+          echo "branch_name=$branch_name" >> $GITHUB_ENV
+          git checkout -b "$branch_name"
+
+      - name: Update Zowe Version in cli/package.json
+        run: |
+          jq ".zoweVersion = \"V$new_version\"" ${{ env.PACKAGE_JSON_PATH }} > package.tmp.json && mv package.tmp.json ${{ env.PACKAGE_JSON_PATH }}
+
+      - name: Run update script
+        run: node ".github/updateReleaseMarkdown.js"
+        env:
+          NEW_VERSION: ${{ env.new_version }}
+
+      - name: Commit and Push Changes
+        run: |
+          git config --global user.name "${{ secrets.ZOWE_ROBOT_USER }}"
+          git config --global user.email "${{ secrets.ZOWE_ROBOT_EMAIL }}"
+          git add "${{ env.PACKAGE_JSON_PATH }}"
+          git add "RELEASE_HISTORY.md"
+          git commit -sm "Update version to ${{ env.new_version }}"
+          git push origin "$branch_name"
+
+      - name: Create Pull Request
+        run: |
+          pr_title="Update CLI version to ${{ env.new_version }}"
+          base_branch="${{ env.defaultBranch }}"
```

Since the `defaultBranch` variable is no longer used after testing, we could use the branch that triggered the workflow as the base branch instead.
```suggestion
          base_branch="${{ github.ref_name }}"
```